packages/nimble-components/src/combobox/tests/combobox.mdx

```diff
@@ -0,0 +1,31 @@
+import {
+    Controls,
+    DocsStory,
+    Meta,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import * as comboboxStories from './combobox.stories';
+
+<Meta of={comboboxStories} />
+
+<Title of={comboboxStories} />
+<Description of={comboboxStories} />
+
+<DocsStory of={comboboxStories.underlineCombobox} expanded={false} />
+<Controls of={comboboxStories.underlineCombobox} />
+
+# Usage Docs
+
+## Combobox value
+
+### Native element and Blazor
+
+The value of the combobox comes from the text content of the selected autocomplete `nimble-list-option`, or, if no matching autocomplete is found, the value is the user-entered text.
+
+### Angular
+
+In Angular, an autocomplete `nimble-list-option` can be created with an `ngValue`. In that case the `ngModel` of the combobox will be the `ngValue` of the matched autocomplete option.
+If no matching autocomplete option is found, the `ngModel` of the combobox will be set to `OPTION_NOT_FOUND`, and if the application needs to access the user-entered text, that should be done
+through the `value` on the `nativeElement` associated with the `nimble-combobox`.
```

Comment:
> that should be done through the `value` on the `nativeElement` associated with the `nimble-combobox`.

I don't think we should ever describe accessing the `nativeElement` as the expected workflow. Is that the case anywhere else? If accessing the value property is an expected workflow it should be exposed on the combobox directive.

Can we expose just a getter for this use case or are there valid use cases to independently set the value as well? I suspect we don't want to independently set the value as then the ngModel wouldn't be aligned. @atmgrifter00

Reply:
Full disclosure @rajsite, I was fine with the current documentation in lieu of getting around the issue raised [here](https://github.com/ni/nimble/issues/682)(particularly regarding the second solution proposal).

I believe your suggestion is ultimately what makes the most sense (and is aligned with the second proposal). Do you think we just want to avoid mentioning `nativeElement` altogether? I believe what prompted this doc change is that we had a client who wanted access to the current text of the `Combobox`, but could find no docs on how to do so. This change is simply meant to offer a temporary relief of that confusion.

Reply:
> Do you think we just want to avoid mentioning nativeElement altogether?

Proper usage of our controls should not necessitate accessing `nativeElement` in Angular.

> and is aligned with the second proposal

The second proposal says to expose the underlying `value` for all controls. I don't think that's appropriate either (too easy to shoot yourself in the foot and screw up ngModel state). It's an exception just for this case. And I think the rework renders this moot as the ngModel value will ultimately just be the text value of the selected option.

So this is just a workaround till the rework. So we could either document it that way and link to bugs or we can expose just the get accessor for value (assuming that is the use case needed today)

Reply:
I've added a `value` getter to the directive and updated these docs.

Reply:
FYI added that this PR fixes the issue meyer's referenced to the description https://github.com/ni/nimble/issues/682

Reply:
Would be nice in a tech debt or something to clean-up SLE to use value

Reply:
I've created a tech debt item in SLE for this and assigned it to myself. I'll take a look at it in a few days, when the new version of nimble should have made its way into apps.